### PR TITLE
Add installation

### PR DESCRIPTION
**Pull request by rentainhe**
## TODO
- Refine https://github.com/Oneflow-Inc/libai/pull/120

### CODE REVIEW

File: docs/source/tutorials/Getting Started.md [hunk 1]
```diff
@@ -1,4 +1,50 @@
...
+    │   ├── img5.jpeg
+    │   └── ...
+    ├── class2
+    │   ├── img6.jpeg
+    │   └── ...
+    └── ...
+
+```
+### Train vit model from scratch
+- Update the data path in [vit_imagenet](https://github.com/Oneflow-Inc/libai/blob/main/configs/vit_imagenet.py) config file
```
Comment (Kitten97) [inline docs/source/tutorials/Getting Started.md:36]:
```suggestion
- Update the data path in [vit_imagenet](https://github.com/Oneflow-Inc/libai/blob/main/configs/vit_imagenet.py) config file:
```

**Comment by Kitten97** [comment]
rest LGTM

File: README.md [hunk 2]
```diff
@@ -24,59 +24,66 @@
...
 
-- **Easy Usage**
+- **Easy to use**
 
-    Components are designed to be modular in LiBai for better and easier usage as follows:
+    LiBai's components are designed to be modular for easier usage as follows:
     - LazyConfig system for more flexible syntax and no predefined structures 
     - Friendly trainer and engine
-    - Used as a library to support building research projects on top of it. Please see [projects/](/projects) for some projects that are built on top of LiBai.
+    - Used as a library to support building research projects on top of it. See [projects/](/projects) for some projects that are built on top of LiBai
```
Comment (wwd190906) [inline README.md:49]:
```suggestion
    - Used as a library to support building research projects on it. See [projects/](/projects) for some projects that are built on LiBai
```

File: README.md [hunk 3]
```diff
@@ -24,59 +24,66 @@
...
-Please refer to [docs](https://libai.readthedocs.io/en/latest/index.html) for full API documentation and tutorials
+
+See LiBai's [documentation](https://libai.readthedocs.io/en/latest/index.html) for full API documentation and tutorials.
 
 ## ChangeLog
 
 **Beta 0.1.0** was released in 15/2/2022:
 - Support 3D parallelism [BERT](https://arxiv.org/abs/1810.04805) models for pretraining.
-- Support 2D Parallelism [ViT](https://arxiv.org/abs/2010.11929) models for Image Classification.
+- Support 2D Parallelism [ViT](https://arxiv.org/abs/2010.11929) models for image classification.
```
Comment (wwd190906) [inline README.md:71]:
```suggestion
- Support 2D parallelism [ViT](https://arxiv.org/abs/2010.11929) models for image classification.
```

**Review by L1aoXingyu — APPROVED** [review]

**Pull request merged**